tests/scripts/basic-in-docker.sh

```diff
@@ -42,6 +42,9 @@ for compiler in clang gcc; do
     run_in_docker -e CC=${compiler} make
     run_in_docker -e CC=${compiler} make test
     run_in_docker programs/test/selftest
+    run_in_docker -e OSSL_NO_DTLS=1 tests/compat.sh
```

Comment:
Note: the end of the commit message should probably read "This only turns up changes in `basic-in-docker.sh`" rather than `run_in_docker.sh`.